test/config.yaml

```diff
@@ -0,0 +1,86 @@
+#### General Settings ####
+# location of system level settings
+settings:
+  title: My Very Cool Project
+  Author: Bob
+
+  # path like settings
+  data: /data/bob/original_data
+
+  # conda environment names
+  python2: py2.7
+
+  # names to access specific envs by
+  env: HOME
```

Comment:
Would `env` have multiple entries?  I think most tools can go in the same env, with the exception of py2-only which you're specifying separately here. So really we just need a py2 env and a py3 env. Actually, once snakemake has per-rule envs this will be obsolete.